src/cloud-api-adaptor/podvm/files/etc/systemd/system/kata-agent.service

```diff
@@ -1,11 +1,9 @@
 [Unit]
 Description=Kata Agent
 BindsTo=netns@podns.service
-Wants=process-user-data.service attestation-agent.service
-After=netns@podns.service process-user-data.service attestation-agent.service
+After=netns@podns.service process-user-data.service
```

Comment:
attestation-agent.service should be there as well, no ?

Reply:
kata-agent is activated by the CDH socket (in the kata-agent.path directive), there is no direct dependency on attestation-agent (indirectly yes, since CDH is depending on the AA socket). It probably wouldn't hurt to leave it in, but since we have to follow a strict ordering of components for sealed secrets, I think it makes sense to be explicit about the dependencies.